CMakeLists.txt

```diff
@@ -56,7 +56,8 @@ set(CMAKE_AUTOMOC ON)
 find_package(Qt5Core 5.0 REQUIRED)
 find_package(Qt5Widgets 5.0 REQUIRED)
 find_package(Qt5Multimedia 5.0 REQUIRED)
-#find_package(QT5OpenGL 5.0 REQUIRED)
```

Comment:
I believe this has been removed to also remove dependency with graphical items at top level. This is inconsistent with the rest of required packages in this list like Widgets and OpenGL itself so it needs more careful revision. I think the intention was to open possibility to compile only a server version.

Reply:
I think we should have a seperate patch (set?) focused on removing top-level dependencies on graphics since that's bound to require much more careful planning than simply fixing the build like this set tries to do.

Reply:
This is still required but the name of the package is wrong. Notice the uppercase `T`. This should be added as `find_package(Qt5OpenGL 5.0 REQUIRED)`.